lib/std/heap.zig

```diff
@@ -815,12 +842,12 @@ pub fn testAllocatorLargeAlignment(base_allocator: mem.Allocator) !void {
     var validationAllocator = mem.validationWrap(base_allocator);
     const allocator = validationAllocator.allocator();
 
-    const large_align: usize = mem.page_size / 2;
+    const large_align: usize = pageSize() / 2;
 
     var align_mask: usize = undefined;
-    align_mask = @shlWithOverflow(~@as(usize, 0), @as(Allocator.Log2Align, @ctz(large_align)))[0];
+    align_mask = @shlWithOverflow(~@as(usize, 0), @as(Allocator.Log2Align, @ctz(@as(u7, @truncate(large_align)))))[0];
 
-    var slice = try allocator.alignedAlloc(u8, large_align, 500);
+    var slice = try allocator.alignedAlloc(u8, 512, 500);
```

Comment:
Why are these changes needed?

Reply:
Line 848, without the changes I get compiler errors

Line 850, I think this is inside a test which uses a large alignment which that depended on the page size which means trying to use runtime stuff at comptime.

Reply:
Probably could move the large_align to runtime with the `memAlign` method?

Reply:
I added the large align to actually align.